src/error.rs

```diff
@@ -1,3 +1,28 @@
 use std::{error, result};
+use std::fmt::Display;
 
 pub type Result<T> = result::Result<T, Box<error::Error>>;
+
+pub trait OrExit {
```

Comment:
@0xd34d10cc haha, I know what is a trait, thanks! :D By the pattern I meant the `OrExit` pattern. "We either have a result or the further executing doesn't make any sense anymore so we quit". It looks very conventional and I'm surprised that something like `OrExit` is not a part of standard library. It can be also implemented by Option for example, or pretty much by anything 'unwrappable'.